scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -285,6 +299,27 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
       {% if not case.causatives|length and not case.partial_causatives|length %}
         No variants marked causative
       {% endif %}
+      <!-- variants with cancer tier I or II -->
+      {% if case.track == 'cancer' %}
+        {% for variant in evaluated_variants %}
+          {% set exp_tiers =  ['1A', '1B', '2C', '2D'] %}
```

Comment:
Also at least P/LP in ACMG right?

Reply:
It would be less confusing to show each variant only once, but with all badges etc. Consider e.g. if there would be a slightly ambiguous DEL call, that is both Tier 1B, LP and marked causative - one could get the impression there were several similar variants.